src/dune/watermarks.ml

```diff
@@ -301,6 +301,42 @@ let get_name ~files ~(dune_project : Dune_project.t option) () =
         ] );
   name
 
+let make_watermark_map_dune_project ~commit ~version ~dune_project =
```

Comment:
Just to keep the code as obvious as possible, I propose to rename the original function `make_watermap_map_from_opam_file` and this function `make_watermap_map_from_dune_project`